src/main/java/com/google/gerrit/extensions/common/ReviewerInfo.java

```diff
@@ -0,0 +1,23 @@
+// Copyright (C) 2014 The Android Open Source Project
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+
+package com.google.gerrit.extensions.common;
+
+public class ReviewerInfo extends AccountInfo{
+  public ApprovalInfo approvalInfo;
```

Comment:
Shouldn't this be a list of ApprovalInfo? I think you could easily verify that this works with a test for the ReviewerInfoParser (there are tests for other parsers around so this should be quite straight forward).